src/main/java/org/traccar/protocol/Gl200TextProtocolDecoder.java

```diff
@@ -122,12 +162,12 @@ private Position decodeAck(Channel channel, SocketAddress remoteAddress, String[
 
     private static final Pattern PATTERN_INF = new PatternBuilder()
             .text("+").expression("(?:RESP|BUFF):GTINF,")
-            .expression("(?:.{6}|.{10})?,")      // protocol version
+            .expression("(.{6}|.{10})?,")      // protocol version
```

Comment:
I believe we should do this for all types, not just this.

Reply:
The other types are not using it.

Reply:
The logic should be common.